en/docs/Learn/Extensions/SAML2SSO/configuring-identity-server-as-idp-for-sso.md

```diff
@@ -118,16 +118,17 @@ For example, let's share a JDBC user store (MySQL) with both the WSO2 Identity S
     !!! note
         To enable tenant-specific SSO with IS 5.9.0 for `API Publisher` and `Developer Portal`, enable **Use tenant domain in local subject identifier** under the Local & Outbound Authentication Configuration section.
 
-    ![enable-tenant-domain](../../../assets/img/Learn/Extensions/SAML2SSO/enable-tenant-domain.png)
+        ![enable-tenant-domain](../../../assets/img/Learn/Extensions/SAML2SSO/enable-tenant-domain.png)
```

Comment:
Use {{base_path}} instead of ../ for all the paths.

Reply:
hanks. fixed in 00819e0